users.js

```diff
@@ -657,6 +657,7 @@ class User {
 
 		if (Tells.inbox[userid]) Tells.sendTell(userid, this);
 		SG.showNews(userid, this);
+		SG.giveDailyReward(userid, this);
```

Comment:
Yeah
this was triggering but if Db does not has the user it sets to 0 which was the problem